src/.example.env

```diff
@@ -4,6 +4,7 @@
 
 # For deployment behind reverse proxy
 APP_BASE_URL=
+PROTOCOL='http'
 
 # humanID Core
 HUMANID_URL=
```

Comment:
I noticed this .env is placed inside the src directory. However, upon checking the .env in the production web-login I found that there is no .env inside the src directory - only the root. Should I create this .env and place these values into it?

Reply:
put it here `/opt/humanid/.web-login.env`

Reply:
the value should be `https` on the server.

Reply:
great thank you!